src/cli/stats-docs.ts

```diff
@@ -0,0 +1,141 @@
+import { Config, LintConfig, loadConfig } from '..';
+import { BaseResolver, Document, resolveDocument } from '../resolve';
+import { detectOpenAPI, OasMajorVersion, openAPIMajor } from '../validate';
+import { normalizeTypes } from '../types';
+import { Oas3Types } from '../types/oas3';
+import { Oas2Types } from '../types/oas2';
+import { WalkContext, walkDocument } from '../walk';
+import { normalizeVisitors, Oas2Visitor, Oas3Visitor } from '../visitors';
+import * as colors from 'colorette';
+import { getFallbackEntryPointsOrExit } from '../cli';
+import { Oas2Parameter } from '../typings/swagger';
+import { Oas3Parameter, OasRef } from '../typings/openapi';
+import { performance } from 'perf_hooks';
+import { getExecutionTime } from '../utils';
+
+interface IStatsRow {
+  metric: string;
+  total: number;
```

Comment:
What if some metric is not a number but e.g. a list of uniqute things or something more advanced?

Let's keep it simple for now but keep in mind we'll need other type of stats later.